website/views/shared/profiles/achievements.jade

```diff
@@ -52,35 +52,35 @@ div(ng-if='::profile.backer.tier')
   small=env.t('kickstartText')
   hr
 
-div(ng-if='profile.achievements.streak || user._id == profile._id')
+div(ng-if='::profile.achievements.streak || user._id == profile._id')
   .achievement.achievement-thermometer(ng-if='::profile.achievements.streak')
   div(ng-class='::{muted: !profile.achievements.streak}')
-    h5(ng-show='profile.achievements.streak > 1 || !profile.achievements.streak')
+    h5(ng-if='::profile.achievements.streak > 1 || ::!profile.achievements.streak')
```

Comment:
Yep, the [angular code](https://githu`bcom/angular/angular.js/blob/cee429f0aaebf32ef1c9aedd8447a48f163dd0a4/src/ng/parse.js#L1027) is just looking for it at the beginning of the `ng-if`, so anytime you're evaluting an `ng-if,` it should just have a single `::` at the beginning of the `ng-if`.